src/styles/styles.js

```diff
@@ -478,6 +476,62 @@ const styles = {
         marginRight: 8,
     },
 
+    expensiTextInputWrapper: {
+        height: 52,
+    },
+    expensiTextInputContainer: {
```

Comment:
Looks style is an _exact_ duplicate of `picker.inputIOS`. So we should collude them into a single reusable style.

Reply:
Where can I define a reusable style?

Reply:
All these styles can be reusable – it's just a question of keeping them generic and giving them generic names. Then they can be composed into more specific styles. This makes it much easier to make major adjustments to the aesthetic in the app in the future, implement multiple themes/modes, etc...